app/views/todos/index.html.erb

```diff
@@ -1,9 +1,14 @@
 <h1>Forgettable TODO</h1>
 
+
+<% if user_signed_in? %>
 <section>
   <%= javascript_pack_tag 'hello_react' %>
 </section>
-
+<% else %>
+  <%= link_to 'Sign up with GitHub', user_github_omniauth_authorize_path %>
+<% end %>
+<!--
```

Comment:
Do not remove whole features for now.
Let's migrate to typescript gradually.

Reply:
Update to remain rails views.